src/main/resources/form_templates.json

```diff
@@ -2788,124 +2788,165 @@
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH10",
   "formName" : "SH10 - Give notice of particulars of variation of rights attached to shares",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH11",
   "formName" : "SH11 - Give notice of a new class of members",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH12",
   "formName" : "SH12 - Give notice of particulars of variation of class rights",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH13",
   "formName" : "SH13 - Give notice of name or other designation of class of members",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH14",
   "formName" : "SH14 - Notify a redenomination of shares",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH15",
   "formName" : "SH15 - Notify a reduction of capital following redenomination",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH16",
   "formName" : "SH16 - Give notice of application to court to cancel special resolution",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
 }, {
   "_id" : "SH17",
   "formName" : "SH17 - Give notice by the company of application to cancel special resolution",
-  "formCategory" : "SH",
+  "formCategory" : "SH-OTHER",
   "fee" : "",
   "isAuthenticationRequired" : true,
   "isFesEnabled" : true,
   "messageTextIdList": [1,4]
-} ]
\ No newline at end of file
+}, {
+  "_id" : "S-CAP-RES",
+  "formName" : "Resolution - Approving a share capital reduction",
+  "formCategory" : "SH-RED",
+  "fee" : "",
+  "isAuthenticationRequired" : true,
+  "isFesEnabled" : true,
+  "messageTextIdList": [1,4]
+}, {
+  "_id" : "CAP-SS",
+  "formName" : "Solvency statement",
+  "formCategory" : "SH-RED",
+  "fee" : "",
+  "isAuthenticationRequired" : true,
+  "isFesEnabled" : true,
+  "messageTextIdList": [1,4]
+}, {
+  "_id" : "SH20",
+  "formName" : "Statement by directors",
+  "formCategory" : "SH-RED",
+  "fee" : "",
+  "isAuthenticationRequired" : true,
+  "isFesEnabled" : true,
+  "messageTextIdList": [1,4]
+}, {
+  "_id" : "SH19",
```

Comment:
Is it correct that SH19 will be listed **after** SH20 in the forms presented?

Reply:
it's this order in the prototype..
I think they're meant to file the SH19 last